bib/east.bib

```diff
@@ -10435,11 +10298,11 @@ @article{east:5410
 }
 
 @Book{east:5411,
-  title      = {rtsod pa'i rigs pa'i 'grel pa don rnam par 'byed pa},
   date       = {1100},
-  translator = {{Kumāraśrībhadra} and {'phags pa shes rab} and {'bro seng dkar (śākya 'od)}},
+  title      = {rtsod pa'i rigs pa'i 'grel pa don rnam par 'byed pa},
   language   = {bo},
-  series     = {snar thang 3717 tshe 21b–131b; snar thang 3770 ze 65b–186a; sde dge 4239 zhe 51a3–151a6; co ne zhe 50b3–143b2; {Peking} 5725 tshe 21b2–137a8; {Peking} 5738 ze 71a5–183a7},
+  series     = {snar thang 3717 tshe 21b2–131b7; snar thang 3770 ze 65b4–186a7; sde dge 4239 zhe 51a3–151a6; co ne zhe 50b3–143b2; {Peking} 5725 tshe 21b2–137a8; {Peking}},
```

Comment:
{Peking} 5738 ze 71a5–183a7 should not be deleted.

Reply:
The second entry in Peking (5738, as you say) was missing from the beginning in EAST, again a typo I think. This corresponds to SUEBS 14.2.T1.
The correct and complete folio numbers thus are: snar thang 3717 tshe 21b2–131b7; snar thang 3770 ze 65b4–186a7; sde dge 4239 zhe 51a3–151a6; co ne zhe 50b3–143b2; {Peking } 5725 tshe 21b2–137a8; {Peking} 5738 ze 71a5–183a7.